Umbraco-Cloud/Troubleshooting/Deployments/index.md

```diff
@@ -36,3 +36,54 @@ The most common Content [Transfer](../../Deployment/Content-Transfer) / [Restore
 If you experience problems when using third-party packages on Umbraco Cloud there is a chance they are not compatible with Umbraco Cloud. Packages that add custom editors will need a Value Connector set up to work with Umbraco Deploy. Some of the most used packages out there have been included in the Community driven Umbraco Deploy Contrib project. You can include the Contrib dll in your project to help with a lot of them.
 
 To see a list of packages covered look [here](https://github.com/umbraco/Umbraco.Deploy.Contrib).
+
+## Issues with .net 7 deployments between local and envs.
+
+If you are experiencing an issue where in your deployment output you are seeing build errors targeting a preview / RC version of .net 7 then you can follow these steps to mitigate the issue. 
```

Comment:
```suggestion
If you are experiencing an issue where your deployment output has build errors targeting a preview / RC version of .NET 7, you can follow these steps to mitigate the issue. 
```

Reply:
I'll add in a note before the steps to resolve the issue it will be resolve in a upcoming release